spec/rubocop/cop/rspec/builtin_matcher_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::BuiltinMatcher do
+  it 'registers an offense when using `be_all`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_all(bar)
+                     ^^^^^^^^^^^ Use `all` instead of `be_all`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to all(bar)
```

Comment:
Is this correct?

Reply:
I have changed to the following correct code:
```ruby
expect(foo).to be_all be_odd
```